sources/packages/web/src/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <modal-dialog-base :showDialog="showDialog" title="StudentAid BC Declaration">
+    <template #content>
+      <p>
+        Please review and accept the StudentAid BC declaration to allow the
+        institution to make proposed changes to your application (on your
+        behalf).
+      </p>
+      <content-group-info class="my-3">
+        <v-row
+          ><v-col
+            ><h4 class="label-bold color-blue">
+              StudentAid BC Declaration
+            </h4></v-col
+          ></v-row
+        >
+        <v-row
+          ><v-col
+            ><h4 class="label-bold">I. I. I/we understand that:</h4>
+          </v-col></v-row
+        ><v-list bg-color="#f5f6f7" class="mb-5">
```

Comment:
we have the color theme and classes required for our application in our `.scss` file. Pls use that